pdns/pdns_recursor.cc

```diff
@@ -2061,7 +2061,7 @@ static void houseKeeping(void *)
     }
 
     if(!t_id) {
-      if(now.tv_sec - last_stat >= 1800) {
+      if(now.tv_sec - last_stat >= ::arg().asNum("statistics-interval") && ::arg().asNum("statistics-interval") != 0) {
```

Comment:
`.asNum()` is pretty expensive. Perhaps a `g_statisticsInterval` would be better?